T3/inputs/skim_lep_tmpl.py

```diff
@@ -14,225 +14,46 @@
 import ROOT as root
 from PandaCore.Tools.Misc import *
 from PandaCore.Tools.Load import *
-import PandaCore.Tools.job_management as cb
+import PandaCore.Tools.job_config as cb
```

Comment:
Why job_config? You cannot import this on non-T3 worker nodes

Reply:
I created the template from skim_wlnhbb_tmpl.py. If this one is problematic, the other one too. Do we have a better example?

Reply:
I created the template from skim_wlnhbb_tmpl.py. If this one is problematic, the other one too. Do we have a better example?

Reply:
I created the template from skim_wlnhbb_tmpl.py. If this one is problematic, the other one too. Do we have a better example?

Reply:
I created the template from skim_wlnhbb_tmpl.py. If this one is problematic, the other one too. Do we have a better example?

Reply:
I created the template from skim_wlnhbb_tmpl.py. If this one is problematic, the other one too. Do we have a better example?

Reply:
Here's what the import should be: https://github.com/sidnarayanan/PandaAnalysis/blob/dev-deep/T3/inputs/skim_wlnhbb_tmpl.py#L17 . @DylanHsu you may want to check what is going on in your code.